backend/LexBoxApi/Services/LexProxyService.cs

```diff
@@ -38,8 +38,12 @@ public class LexProxyService : ILexProxyService
         return user;
     }
 
-    public async Task RefreshProjectLastChange(string projectCode)
+    public async Task RefreshProjectLastChange(string projectCode, TimeSpan? delay = null)
```

Comment:
You don't seem to be using the `delay` parameter.
```suggestion
    public async Task RefreshProjectLastChange(string projectCode)
```